handlers/services.go

```diff
@@ -71,13 +71,21 @@ func getServiceMetrics(w http.ResponseWriter, r *http.Request, promClientSupplie
 			return
 		}
 	}
+	var byLabelsIn []string
+	var byLabelsOut []string
+	if lblsin, ok := queryParams["byLabelsIn[]"]; ok && len(lblsin) > 0 {
```

Comment:
I guess yes, but I need to ask, the "byLabelsIn[]" the "[]" is intended ?

Reply:
yes, this is how axios generated my array query params, and I could get them this way in go, working well so I didn't looked further.